src/core/mod.rs

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2020, Arm Limited, All Rights Reserved
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//          http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#![allow(dead_code)]
+
+mod request_handler;
+
+use crate::auth::AuthToken;
+use derivative::Derivative;
+use parsec_interface::operations::{Convert, NativeOperation, NativeResult};
+use parsec_interface::operations_protobuf::ProtobufConverter;
+use parsec_interface::requests::{
+    request::RequestHeader, BodyType, ProviderID, Request, Response, ResponseStatus, Result,
+};
+use request_handler::RequestHandler;
+
+/// OperationHandler structure to send a `NativeOperation` and get a `NativeResult`.
+#[derive(Derivative)]
+#[derivative(Debug)]
+pub struct OperationHandler {
```

Comment:
If the Core operations currently in `src/lib.rs` went here, this file could go to `src/core/operation_handler.rs` for example.

Reply:
Makes sense!